packages/sdk/utils/src/types.d.ts

```diff
@@ -1,4 +0,0 @@
-declare module '@umpirsky/country-list/data/es_AR/country.json' {
```

Comment:
@gastonponti do we still need this in types just from a different source?

Reply:
We could, but the way is used, it's the same.
We were not using that as we should, and with the `require` part is assuming the object as any, which will also work with the `esData[...]` thing, without requiring that types declaration.
We can change that using imports, but we will require some changes in the Countries class to fully use the typing.

I think that due to the scope of this PR, it's easier to leave it this way, and then fixing/force types in another issue if we need that